src/lucky/form_data.cr

```diff
@@ -0,0 +1,45 @@
+class Lucky::FormData
+  getter params = MultiValueStorage(String).new
+  getter files = MultiValueStorage(Lucky::UploadedFile).new
+
+  def add(part : HTTP::FormData::Part)
+    case part.headers
+    when .includes_word?("Content-Disposition", "filename")
+      files.add(part.name, Lucky::UploadedFile.new(part))
+    else
+      params.add(part.name, part.body.gets_to_end)
+    end
+  end
+
+  # Simpler, generic implementation of HTTP::Params
+  class MultiValueStorage(T)
+    include Enumerable({String, T})
+
+    private getter storage : Hash(String, Array(T))
+
+    def initialize
+      @storage = {} of String => Array(T)
+    end
+
+    def ==(other : self)
+      self.storage == other.storage
```

Comment:
this will fail because `storage` is private. You could make it `protected`, or public is probably ok?

Reply:
Let me just remove it. We don't use it anywhere, it was copied from the HTTP::Params class and I don't know why i felt like keeping it at the time.